src/api/data/cache/charthelper/chart_package_helper.go

```diff
@@ -134,6 +136,19 @@ var ensureChartDataDir = func(chart *models.ChartPackage) error {
 	return nil
 }
 
+var cleanChartDataDir = func(chart *models.ChartPackage) error {
```

Comment:
why is this a variable? do you intend to replace it?

or do you do that for every function just in case?